### PR TITLE
VIMC-3421 remove annex as foreign server

### PR DESCRIPTION
**Pull request by hillalex**

### CODE REVIEW

File: scripts/start.sh [hunk 1]
```diff
@@ -75,7 +75,4 @@ docker exec $DB_ANNEX_CONTAINER montagu-wait.sh
 docker run --rm --network=$NETWORK $MIGRATE_IMAGE -configFile=conf/flyway-annex.conf migrate
```
Comment (richfitz) [inline scripts/start.sh]: Should we be bringing up annex at all in this script now? Similar issues in scripts/stop.sh and scripts/test-migrate.sh

File: migrations/sql/V2020.02.10.1834__RemoveAnnex.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+DROP SERVER montagu_db_annex CASCADE;
```
Comment (richfitz) [inline migrations/sql/V2020.02.10.1834__RemoveAnnex.sql]: Add `DROP EXTENSION postgres_fdw` too?

File: scripts/start.sh (hunk 1 above)
Comment (hillalex) [reply scripts/start.sh]: Do we know where this script is in use? It is used by the api, but anywhere else?

Comment (richfitz) [reply scripts/start.sh]: that's all I can fine

**Pull request merged**